.github/workflows/regenerate-sdks.yml

```diff
@@ -1,209 +0,0 @@
-on:
-  workflow_dispatch: {}
-env:
-  PROVIDER: command
-  GITHUB_TOKEN: ${{ secrets.PULUMI_BOT_TOKEN }}
-  GOVERSION: 1.21.x
-  NODEVERSION: 16.x
-  PYTHONVERSION: "3.7"
-  DOTNETVERSION: |
-    6.0.x
-    3.1.301
-  JAVAVERSION: "11"
-
-jobs:
-  build_provider:
```

Comment:
This workflow seems like it was completely unused and save to remove (no previous runs shown in https://github.com/pulumi/pulumi-command/actions)